docker-compose.yml

```diff
@@ -5,8 +5,6 @@ services:
     container_name: apostrophe
     restart: always
     build: .
-    ports:
-      - "3000:3000"
```

Comment:
Wait, how does port 3000 expose mongo? That's for apostrophe.

Reply:
Yikes, totally my bad. Meant to check if Mongo was being exposed. 🤦🏻